modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,75 @@
 [id="configuring-default-seccomp-profile_{context}"]
-= Configuring the default seccomp profile
-OpenShift ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` as following:
+= Enabling the default seccomp profile for all pods
```

Comment:
Shouldn't the file start with some metadata, including the content type attribute?
"Each .adoc file must contain a content type attribute in the metadata: :_content-type: <TYPE>"